### PR TITLE
fixed to read "serial" field in PDB files.

### LINKED ISSUE #12619: PDBLoader does not read "serial" field in the PDB file
##### Description of the problem 

When reading a PDB file, this loader ignore "serial" field (col : 7-11).
So, atoms are not connected correctly if ATOM/HETATM records are not shown in the PDB file orderly.

example data:
----
COMPND  H2O.PDB                                                                 
HETATM    2  H                   0.735  -0.088  -0.015                
HETATM    3  H                  -0.724  -0.347  -0.042                
HETATM    1  O                  -0.095   0.353  -0.075                
CONECT    1    2    3                                        
CONECT    2    1                                             
CONECT    3    1                                             
END   
----

##### Three.js version

- [ ] Dev
- [x] r87
- [ ] ...

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

### PR DESCRIPTION
**Pull request by kenkiku1021**
This PR fix #12619 
To read serial field in PDB files.

### CODE REVIEW

File: examples/js/loaders/PDBLoader.js [hunk 1]
```diff
@@ -174,7 +175,7 @@ THREE.PDBLoader.prototype = {
 
 				}
 
-				atoms.push( [ x, y, z, CPK[ e ], capitalize( e ) ] );
+				atoms[index] = [ x, y, z, CPK[ e ], capitalize( e ) ];
```
Comment (Mugen87) [inline examples/js/loaders/PDBLoader.js]: Code style: `atoms[ index ]`.

**Review by Mugen87 — CHANGES_REQUESTED** [review]

**Comment by Mugen87** [comment]
The existing example works with these changes: http://rawgit.com/kenkiku1021/three.js/PDBLoader-fix/examples/webgl_loader_pdb.html

**Comment by Mugen87** [comment]
See http://www.wwpdb.org/documentation/file-format-content/format33/sect9.html#ATOM

@kenkiku1021 I guess the loader parses now this section, right?

```
 7 - 11        Integer       serial       Atom  serial number.
```

**Comment by kenkiku1021** [comment]
@Mugen87 That's right. The loader should parse the serial section.

**Comment by kenkiku1021** [comment]
I fixed coding style for 'three.js'.

**Comment by Mugen87** [comment]
Looks good! 👍

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**